packages/wallet/backend/migrations/20230711171065_add_password_reset_fields.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.alterTable('users', (table) => {
+    table.boolean('isEmailVerified').defaultTo(false)
```

Comment:
Should we handle the use-case in which we already have users that are registered and set the `isEmailVerified` column to `true` ?

Reply:
I thought that we will start from scratch and it is not needed, but it won't hurt doing it